LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 DV CORREIA
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
```

Comment:
- [ ] Hey kudos to you ,you did a great job on this milestone,however there are some little things you omitted to address on your project readme .
- [ ] According to the project requirement The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).Please kindly include this as Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).

The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project. please fix this and resubmit your project

Reply:
Thank you @ademibowale. I added this to the readme, and will send for another review!

Reply:
@dvcorreia1984 Thats great and thank you for accepted my  changes request.